### DISCUSSION TITLE
Brazilian disc circular boundary generation in Neper

### DISCUSSION THREAD

**Discussion by Ali141272** [neperfepx/neper#495] [Q&A] [answered]
Hello everyone, 
How to generate a circular boundary without sharp edges for the Brazilian disc?
In Neper I used this command neper -T -n 15 -morpho gg -domain "circle(0.03)" -dim 2 and built it but it has sharp edges. I need it without sharp edges, please have look at the attachment.
Thank you.
<img width="1070" alt="a15" src="https://user-images.githubusercontent.com/102291665/173404717-81a606c2-0fc5-4eb7-983b-e22f9d13d76a.png">

**Comment by rquey** [comment, marked as answer]
Use the second argument of [-domain "circle(...)"](https://neper.info/doc/neper_t.html#cmdoption-domain) to specify a greater number of edges to discretize the domain boundary:

```
neper -T -n 15 -morpho gg -domain "circle(0.03,100)" -dim 2
neper -V n15-id1.tess -dataedgerad 0.0001 -print img
```
![image](https://user-images.githubusercontent.com/2659376/173420152-785b6593-bf16-4108-b6bf-f31cf34edff5.png)

**Reply by Ali141272** [reply]
Romain Quey, Thank you very much for your kind response.
